common/persistence/cassandra/cassandraQueue.go

```diff
@@ -225,6 +287,36 @@ func (q *cassandraQueue) DeleteMessagesBefore(messageID int) error {
 	return nil
 }
 
+func (q *cassandraQueue) DeleteMessagesFromDLQ(
+	messageID int,
+) error {
+
+	// Use negative queue type as the dlq type
+	query := q.session.Query(templateDeleteMessagesFromDLQQuery, -q.queueType, messageID)
+	if err := query.Exec(); err != nil {
+		return &workflow.InternalServiceError{
+			Message: fmt.Sprintf("DeleteMessagesFromDLQ operation failed. Error %v", err),
+		}
+	}
+
+	return nil
+}
+
+func (q *cassandraQueue) RangeDeleteMessagesFromDLQ(
+	messageID int,
+) error {
+
+	// Use negative queue type as the dlq type
+	query := q.session.Query(templateDeleteMessagesQuery, -q.queueType, messageID)
```

Comment:
this doesn't really do a range delete. Do you even need this method ?

Reply:
renamed to DeleteMessagesFromDLQ. No place is calling this method. The plan is to provide a CLI as purge all messages before <message ID> and it will use this method